install/kubernetes/cilium/charts/agent/templates/clusterrole.yaml

```diff
@@ -18,6 +18,7 @@ rules:
   - services
   - nodes
   - endpoints
+  - secrets
```

Comment:
We can't blindly add this here, check my comment to make this conditional.

Reply:
Done.